Source/DataTypes/SvgUnit.cs

```diff
@@ -164,6 +164,14 @@ public float ToDeviceValue(ISvgRenderer renderer, UnitRenderingType renderType,
                         case UnitRenderingType.VerticalOffset:
                             _deviceValue = (size.Height / 100) * value + boundable.Location.Y;
                             break;
+                        case UnitRenderingType.Other:
+                            // Calculate a percentage value of the normalized viewBox diagonal length. 
+                            if (owner.OwnerDocument != null && owner.OwnerDocument.ViewBox != null && owner.OwnerDocument.ViewBox.Width!=0 && owner.OwnerDocument.ViewBox.Height != 0)

```

Comment:
nitpicking - use the same space formatting for all, e.g. always spaces arounf the operators